src/utils/init/utils.js

```diff
@@ -0,0 +1,136 @@
+const path = require('path')
+
+const chalk = require('chalk')
+const cleanDeep = require('clean-deep')
+const dotProp = require('dot-prop')
+const inquirer = require('inquirer')
+const isEmpty = require('lodash/isEmpty')
+
+const { fileExistsAsync, writeFileAsync } = require('../../lib/fs')
+
+const normalizeDir = ({ siteRoot, dir, defaultValue }) => {
+  if (dir === undefined) {
+    return defaultValue
+  }
+
+  const relativeDir = path.relative(siteRoot, dir)
+  return relativeDir || defaultValue
+}
+
+const getDefaultSettings = ({ siteRoot, config }) => {
+  const defaultBuildCmd = dotProp.get(config, 'build.command')
+  const defaultBuildDir = dotProp.get(config, 'build.publish')
+  const defaultFunctionsDir = dotProp.get(config, 'build.functions')
```

Comment:
`build` is always defined when returned by `@netlify/config`:

https://github.com/netlify/build/blob/60ee6cfb5af57e4fed70fce7bc566d086ce4d653/packages/config/src/normalize.js#L13-L16

So we should be able to access those properties without using `dotProp`.

Reply:
Done in https://github.com/netlify/cli/pull/1619/commits/8af5304350d8fe7db46dd32698694140338368a7